### ISSUE TITLE
Can I use it with preload xml content?

### ISSUE THREAD

**Issue by williamfzc** [aconrad/pycobertura#102]
Hi there, good project!

In my cases, I used some snippets to convert lcov xml to Cobertura XML. And it will generate xml-format string.

```python
converter = LcovCobertura(info_content)
cobertura_xml = converter.convert()

# now I have a xml-format string
```

So can we directly use xml-format string (rather than write it to file and read from it in pycobertura)?
Or if you prefer a PR? :)

**Comment by aconrad** [comment]
I'm sure we can change that in some way to to prevent you from doing this intermediary step. It wasn't designed to support this use case but I'm sure we could accommodate without too much fuss. Have you looked at the code to see if there are opportunities for improvement?

**Comment by williamfzc** [comment]
> Have you looked at the code to see if there are opportunities for improvement?

yep, I think it may be not that hard. (it looks like)

https://github.com/aconrad/pycobertura/blob/cdce1ae0eacd6d8b2954d289a2379330967abf97/pycobertura/cobertura.py#L40

xml content will be parsed at the beginning of `__init__`. We can add a pre-check here to identify if it is a valid xml-string. If it is not, treat it as a path.

**Comment by aconrad** [comment]
Thanks for this @williamfzc! Can you please submit a PR so we follow the standard code reviewing process?

**Comment by williamfzc** [comment]
> Thanks for this @williamfzc! Can you please submit a PR so we follow the standard code reviewing process?

Sure :) see #103

**Comment by aconrad** [comment]
Addressed in #103, closing.
